cli/src/commands/edit.rs

```diff
@@ -21,10 +21,12 @@ use crate::cli_util::{CommandHelper, RevisionArg};
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Edit a commit in the working copy
+/// Loads the contents of the specified revision and sets it as the active changeset.
```

Comment:
What do you mean by "Loads the contents"?

I think "Points the working copy to the specified revision" or "Sets the specified revision as the working-copy commit" make sense.

Reply:
Alrighty. I think I'll use the second one.